util/OptionsDB.cpp

```diff
@@ -137,8 +137,7 @@ OptionsDB::OptionsDB() {
     s_options_db = this;
 }
 
-void OptionsDB::Commit()
-{
+void OptionsDB::Commit() {
```

Comment:
It should be fixed in a separate grooming commit. I'd be fine with it in the pull request, but so far as I understand his position, @adrianbroher perfers to keep grooming out of pull requests to make it easier to review what they actually substantially changed.